docs/getting-started/first-app.md

```diff
@@ -66,33 +66,33 @@ All changes are saved automatically.
 
 ### Create the Description Field
 
-1. In the *Entity Fields* text box type “Description”.
+1. In the _Entity Fields_ text box type “Description”.
 2. Click **Add field** (or just press Enter). The new field is added to the list.
-3. The field's property panel opens. Leave the defaults (*Required Field* and *Searchable* inactive, *Data Type* "Multi Line Text", and *Max Length* "1000".)
+3. The field's property panel opens. Leave the defaults (_Required Field_ and _Searchable_ inactive, _Data Type_ "Multi Line Text", and _Max Length_ "1000".)
 
 ### Create the Start Date Field
 
-1. In the *Entity Fields* text box type “Start Date”.
+1. In the _Entity Fields_ text box type “Start Date”.
 
 :::info
 White spaces are supported, which is useful when you want to enter a descriptive field name. The value is saved as the field’s display name.
-In addition to the display name, each field has an auto-generated *Name* that does not contain spaces or special characters. This name is later used for the API endpoint and in other places in the generated code.
+In addition to the display name, each field has an auto-generated _Name_ that does not contain spaces or special characters. This name is later used for the API endpoint and in other places in the generated code.
 If needed, you can manually change the field name in the field's properties panel.
 :::
 
 2. Click **Add field** (or just press Enter). The new field is added to the list.
-3. The field's property panel opens. Leave the defaults (*Required Field* and *Searchable* inactive, *Data Type* "Date Time", and *Time Zone* "Local Time".)
+3. The field's property panel opens. Leave the defaults (_Required Field_ and _Searchable_ inactive, _Data Type_ "Date Time", and _Time Zone_ "Local Time".)
 
 ### Create the Owner Field
 
-1. In the *Entity Fields* text box type “Owner”.
+1. In the _Entity Fields_ text box type “Owner”.
 2. Click **Add field** \(or just press Enter\). The new field is added to the list.
-3. The field's property panel opens. Change the *Data Type* from "Single Line Text" to "Relation to another Entity".
-4. In the *Related Entity Id* field select “User”.
+3. The field's property panel opens. Change the _Data Type_ from "Single Line Text" to "Relation to another Entity".
+4. In the _Related Entity Id_ field select “User”.
 5. Click the **Required Field** toggle to make the “Owner” field required.
 
 :::tip
-You might find it easier to first add all the fields you want, one after onother, and only afterwards set the properties of each field. 
+You might find it easier to first add all the fields you want, one after another, and only afterwards set the properties of each field.
```

Comment:
@GalCegla good on you for catching that typo! @yuval-hazaz how about the spell-checker action we discussed a while back?

Reply:
unfortunately, I really don't have the capacity currently. For now, use a local spell checker (browser plugins or something similar), I will discuss with @iddan to see what can be done automatically in the pipe, and then I will prioritize.

Reply:
Added a validation action with this commit https://github.com/amplication/docs/commit/7cdb48eb4efb27f295fb604fb28f47c48f99aefa

Reply:
@iddan how do I use it?